setup.py

```diff
@@ -45,8 +45,7 @@ def run(self):
         name="genomekit",
         packages=find_packages(include=["genome_kit"]),
         project_urls={
-            # TODO check documentation link when docs are published on readthedocs
-            "Documentation": f"https://genomekit.readthedocs.io/en/{version}"
+            "Documentation": "https://deepgenomics.github.io/GenomeKit"
```

Comment:
don't we need to version the docs?

Reply:
You're right. This is a non-trivial task on Github Pages, since there's no API for just adding files to an existing site. There's also no east way (afaict) to download the existing site's archive.

I can think of a few ways around this, I'll give it a bit more thought.

Reply:
I'm going to punt on this for now. Any solution I can think of is a significant amount of work.